src/controller/java/AndroidDeviceControllerWrapper.cpp

```diff
@@ -635,6 +635,7 @@ CHIP_ERROR AndroidDeviceControllerWrapper::UpdateAttestationTrustStoreBridge(job
 
     deviceAttestationVerifier = new Credentials::DefaultDACVerifier(attestationTrustStoreBridge);
     VerifyOrExit(deviceAttestationVerifier != nullptr, err = CHIP_ERROR_NO_MEMORY);
+    deviceAttestationVerifier->EnableVerboseLogs(true);
```

Comment:
Could you add a similar bit to `src/darwin/Framework/CHIP/MTRDeviceController_Concrete.mm`?  Or let me know and I will.  Presumably both where it does `new chip::Credentials::PartialDACVerifier()` and where it does `new chip::Credentials::DefaultDACVerifier(trustStore)`.